src/parthenon_manager.cpp

```diff
@@ -222,15 +230,13 @@ void ParthenonManager::RestartPackages(Mesh &rm, RestartReader &resfile) {
     }
   }
   std::vector<Real> tmp(static_cast<size_t>(nb) * nCells * vlen);
-  std::cout << "SIZES:" << nb << ":" << vlen << ":"
-            << static_cast<size_t>(nb) * nCells * vlen << std::endl;
   for (auto &v : ciX.vars) {
     const size_t v4 = v->GetDim(4);
     const std::string vName = v->label();
 
-    std::cout << "Var:" << vName << ":" << v4 << std::endl;
+    if (Globals::my_rank == 0) std::cout << "Var:" << vName << ":" << v4 << std::endl;
```

Comment:
is this a debug statement that should be removed?

Reply:
I put it in there intentionally.  I wanted to give out a status reading that variables are read in from restart file so that the outputs from a direct run and a restart run are evident, and also so that somebody can check to see what variables came in from the restart file.

Reply:
Ah, okay. Got it.